packages/base-zone/src/prepare-revocable.js

```diff
@@ -95,7 +105,7 @@ export const prepareRevocableKit = (
     }),
     {
       revoker: {
-        revoke() {
+        revokeOne() {
```

Comment:
Is this maybe better named `revokeOnce`?

Reply:
Changed to `revoke`. Done.